docs/source/tutorial/understanding_evaluation.rst

```diff
@@ -24,14 +24,94 @@ triple :math:`(h, r, t) \in \mathcal{T}_{eval}` in this set, two tasks are solve
 prediction (left/right), there exist several aggregation measures to quantify the performance of a model in a single
 number.
 
+Rank-Based Metrics
+~~~~~~~~~~~~~~~~~~
+Given the set of individual rank scores $\mathcal{I}$, the following scores are commonly used as aggregation.
+
+Hits @ K
+********
+The hits @ k describes the fraction of true entities that appear in the first $k$ entities of the sorted rank list.
+It is given as:
+
+.. math::
+
+    \text{score}_k = \frac{1}{|\mathcal{I}|} \sum \limits_{r \in \mathcal{I}} \mathbb{I}[r \leq k]
+
+For example, if Google shows 20 results on the first page, then the percentage of results that are relevant is the
+hits @ 20. The hits @ k, regardless of $k$, lies on the $[0.0, 1.0]$ where closer to 1 is better.
+
+.. warning::
+
+    This metric does not differentiate between cases when the rank is larger than $k$.
+    This means that a miss with rank $k+1$ and $k+d$ where $d \gg 1$ have the same
+    effect on the final score. Therefore, it is less suitable for the comparison of different
+    models.
+
+Mean Rank
+*********
+The mean rank (MR) computes the arithmetic mean over all individual ranks. It is given as:
+
+.. math::
+
+    \text{score} =\frac{1}{|\mathcal{I}|} \sum \limits_{r \in \mathcal{I}} r
+
+It has the advantage over hits @ k that it is sensitive to any model performance changes, not only what occurs
+under a certain cutoff and therefore reflects average performance. It the mean rank lies on the
+range $[0.0, \infty)$ where closer to 0 is better.
+
+.. warning::
+
+    While it remains interpretable, the mean rank is dependent on the number of candidates.
+    A mean rank of 10 might indicate strong performance for a candidate set size of 1,000,000,
+    but incredibly poor performance for a candidate set size of 20.
+
+Adjusted Mean Rank
+******************
+The adjusted mean rank (AMR) was introduced by [berrendorf2020]_. It is defined as
+
+.. math::
+
+    \text{score} = \frac{MR}{\mathbb{E}\left[MR\right]} = \frac{2 \sum_{i=1}^{n} r_{i}}{\sum_{i=1}^{n} (|\mathcal{S}_i|+1)}
+
+It lies on the range $[0.0, 2.0]$ where closer to 0 is better.
+
+Adjusted Mean Rank Index
+************************
+The adjusted mean rank index (AMRI) was introduced by [berrendorf2020]_ to make the AMR
+more intuitive.
+
+.. math::
+
+    \text{score} = 1 - \frac{MR - 1}{\mathbb{E}\left[MR - 1\right]} = \frac{2 \sum_{i=1}^{n} (r_{i} - 1)}{\sum_{i=1}^{n} (|\mathcal{S}_i|)}
+
+The AMR has a bounded value range of $[-1, 1]$ where closer to 1 is better.
+
+Mean Reciprocal Rank
+********************
+The mean reciprocal rank (MRR) is the inverse of the harmonic mean of the ranks. It is defined as:
+
+.. math::
+
+    \text{score} =\frac{1}{|\mathcal{I}|} \sum_{r \in \mathcal{I}} r^{-1}
+
+.. warning::
+
+    The mean reciprocal rank has been shown to have serious flaws by [fuhr2018]_. Do **not** rely on it in your work.
+    It is only implemented and reported in PyKEEN for posterity.
```

Comment:
The recommendation by Fuhr is not undisputed, e.g., http://sigir.org/wp-content/uploads/2020/06/p14.pdf. Thus, I would not put such a strong formulation into our documentation.

Reply:
sad, spread the love 2021 is so overrated

Reply:
cf. https://github.com/pykeen/pykeen/compare/3af5349..24e5b9f